app/src/main/java/org/supla/android/NavigationActivity.java

```diff
@@ -459,6 +464,8 @@ public void onClick(View v) {
 
             openHomepage();
 
+        } else if ( v == EmptySpaceButton ) {
```

Comment:
Is this really needed if there is no code inside ?

Reply:
Right, it is unneeded. I deleted it.